tests/utils.py

```diff
@@ -169,6 +169,12 @@ def create_dummy_data_dict(self, path_to_dummy_data, data_url):
             else:
                 value = os.path.join(path_to_dummy_data, abs_path.split("/")[-1])
             dummy_data_dict[key] = value
+
+        # make sure that values are unique
+        if len(set(dummy_data_dict.values())) < len(dummy_data_dict.values()):
```

Comment:
Another hack :-/ to make a new weird download pattern work with dummy_data @mariamabarham @thomwolf @lhoestq @jplu